tests/Sam/PhpFpmRuntimeTest.php

```diff
@@ -47,7 +47,7 @@ public function test error_log function()
     {
         $response = $this->invoke('/?error_log=1');
 
-        self::assertSame(500, $response->getStatusCode(), $this->logs);
+        $this->assertResponseSuccessful($response);
```

Comment:
This is good: using `error_log()` will no longer crash the response by forcing a 500 blank page.